assets/app/features/upgrade/templates/upgrade7/landing.jade

```diff
@@ -1,13 +1,34 @@
-uib-alert(type="danger", ng-if="Upgrade7LandingVm.prechecks.errors")
-  h4(translate='') upgrade7.errors.prechecks.title
-  span(translate='') upgrade7.errors.prechecks.description
-  ul
-    li(ng-repeat="error in Upgrade7LandingVm.prechecks.errors", translate='upgrade7.errors.codes.{{::error}}')
-  button.btn.btn-default.btn-sm(translate='', ng-click="Upgrade7LandingVm.prechecks.runPrechecks()") upgrade7.errors.prechecks.form.check-again
+  
 .panel.panel-default.upgrade-flow
   .panel-heading
     h1(translate='upgrade7.title') Suse Upgrade
   .panel-body
     p(translate='') upgrade7.steps.landing.description
-    p(translate='') upgrade7.steps.landing.description2
-    button.btn.btn-success.center-block.btn-lg(translate='', ng-class="{disabled: !Upgrade7LandingVm.prechecks.completed || Upgrade7LandingVm.prechecks.errors}", ng-click="Upgrade7LandingVm.beginUpgrade()") upgrade7.steps.landing.form.button
+    ul.requirements-list
+      li(translate='') upgrade7.steps.landing.requirements.req01
```

Comment:
Oh, just realise those are the description paragraphs in the mockup.
I think it's confusing the difference between those requirements and the checks. Could we change it to something like `.landing.description.title` and `.landing.description.p1`, `.p2`, `.p3`? What do you think?